### ISSUE TITLE
Nix Manual SEO is awful

### ISSUE THREAD

**Issue by a-kenji** [NixOS/nix#7081]
## Problem

When searching the manual on a search engine, it is almost impossible to surface.
Searching for `nix manual` is not on the first page.

## Checklist

- [x] checked [latest Nix manual] \([source])
- [x] checked [open documentation issues and pull requests] for possible duplicates

[latest Nix manual]: https://nixos.org/manual/nix/unstable/
[source]: https://github.com/NixOS/nix/tree/master/doc/manual/src
[open documentation issues and pull requests]: https://github.com/NixOS/nix/labels/documentation

## Proposal

--- 
Co-authored-by: @mightyiam

**Comment by SuperSandro2000** [comment]
Google shows the right result right away 

![image](https://user-images.githubusercontent.com/7258858/191927649-c47dfcd4-a772-4c8f-9913-4130153ac6b4.png)

**Comment by mightyiam** [comment]
@SuperSandro2000 yeah nah yeah. Look again. In your screenshot the first result is the NixOS manual. The second is the nixpkgs manual. The third is home-manager manual and the fourth seems like some non-canonical publish of the Nix manual.

Causing some confusion is https://github.com/NixOS/nixpkgs/issues/192582.

**Comment by edolstra** [comment]
When I search for `nix manual`, https://nixos.org/manual/nix/stable/introduction.html is the second result.

**Comment by mightyiam** [comment]
Are those personalized results, @edolstra ?

**Comment by edolstra** [comment]
No, that's also in private mode.

**Comment by mightyiam** [comment]
Oh. Well, now I see that that's the case for me, as well. But the title of the search result is misleading: "Introduction - NixOS":
![2022-09-27_10-03](https://user-images.githubusercontent.com/635591/192497344-142de85f-4a5a-4973-ac44-fe5bf3318318.png)

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
